README.md

```diff
@@ -1296,6 +1296,45 @@ _You can enable the following settings in Xcode by running [this script](resourc
   
   </details>
 
+* <a id='extension-access-control'></a>(<a href='#extension-access-control'>link</a>) **Specify the access control for each declaration in an extension individually.** [![SwiftFormat: extensionAccessControl](https://img.shields.io/badge/SwiftFormat-anyObjectProtocol-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md##extensionaccesscontrol)
```

Comment:
Will this rule automatically move access control from before the extension to each member?

Reply:
Yes! Here's an example of what the SwiftFormat rule will do:

```diff
- public extension Foo {
-     func bar() {}
-     func baaz() {}
-     internal func quux() {}
  }

+ extension Foo {
+     public func bar() {}
+     public func baaz() {}
+     func quux() {}
  }
```